### ISSUE TITLE
ePub validation warning: Token '0%' not allowed here

### ISSUE THREAD

**Issue by ashutosh1290** [IDPF/epubcheck#488]
Hi,
I have created one epub and used following css3 features in epub:

@-webkit-keyframes fade-in {
0% {
opacity: 0;
}
100% {
opacity: 1;
}
}

While epub is validating through epub checker 3.0 then checker is generating warning:
"WARNING: mimetype.epub/OPS/assets/css/main.css(696): Token '0%' not allowed here"

Please suggest how can fix this warning.

Thanks,
Ashu

**Comment by tofi86** [comment]
This needs to be reviewed as the original request was based upon EpubCheck 3.0

**Comment by tofi86** [comment]
The current EpubCheck version 4.0.2 and the current development version do not throw this error anymore.

I tested with a basic EPUB and the following CSS snippet:

```css
@-webkit-keyframes fade-in {
  0% { opacity: 0; }
  100% { opacity: 1; }
}
@keyframes fade-in {
  0% { opacity: 0; }
  100% { opacity: 1; }
}
```
